mono/mini/debugger-agent.c

```diff
@@ -4470,6 +4637,29 @@ set_breakpoint (MonoMethod *method, long il_offset, EventRequest *req, MonoError
 	method_seq_points = g_ptr_array_new ();
 
 	mono_loader_lock ();
+#ifdef IL2CPP_MONO_DEBUGGER
+	void *seqPointIter = NULL;
+	Il2CppSequencePointC *seqPoint;
+	while(seqPoint = il2cpp_get_sequence_points(&seqPointIter))
+	{
+		if (bp_matches_method(bp, *(seqPoint->method)) && seqPoint->ilOffset == bp->il_offset)
+		{
+
+			BreakpointInstance* inst = g_new0(BreakpointInstance, 1);
+			inst->il_offset = bp->il_offset;// it.seq_point.il_offset;
+			inst->native_offset = 0;// it.seq_point.native_offset;
+			inst->ip = (guint8*)inst;// (guint8*)ji->code_start + it.seq_point.native_offset;
```

Comment:
Why do we store the inst in the ip?

Reply:
This is from the hackweek debugger. It looks like we don't use the `ip` field for IL2CPP, so I'll remove it and leave it with a `NULL` value.